src/alr/alr-commands-toolchain.adb

```diff
@@ -159,7 +159,7 @@ package body Alr.Commands.Toolchain is
 
          if Origin_Status = Mixed then
             Warnings.Warn_Once
-              ("Default toolchain contains mixed-procedence tools");
+              ("Default toolchain contains mixed-precedence tools");
```

Comment:
Mistranslation of Spanish "procedencia". Either "provenance" or "origin" would work here.